docs/05-engineering/security-incidents.md

```diff
@@ -0,0 +1,49 @@
+# Security incidents
+
+Something went "bump" in the night (or the day)? This document explains what to do when responding to a security incident. See [What is an incident?](#what-is-an-incident) if you need help determining whether something counts as an incident.
+
+## Reporting phishing emails
+If you receive a phishing email, follow these steps to report to CivicActions IT:
+
+1. Do not click any links in the email. Do not delete it yet. You may mark it as spam.
+1. If you can, click the `Show Original` option in the "triangle" menu associated with the email. This will open a new window with the Original Message above and the raw text of the email below.
+1. Click on the `Download Original` link halfway down the page and it will save text of the email to your hard drive.
+1. Mark the email as a phishing email by selecting the `Report phishing` option in the same "triangle" menu associated with the email.
+1. Forward the email to [support@civicactions.com](mailto:support@civicactions.com). As long as you haven't clicked on link or downloaded the file, you may stop here.
+   * If you suspect that the email has compromised your system with a download or link, attach the original text you downloaded. Please include *Security Incident* in the subject line, along with a brief description of the issue (Ex. Clicked on link in phishing email).
+1. Report the phishing email in the [#general](https://civicactions.slack.com/messages/general/) Slack channel (if you have access).
+1. After receiving your notification to support, IT will create a ticket and may contact you for more information.
+
+You might be tempted to simply mark phishing emails as spam and otherwise ignore them, but if you accidentally (or intentionally) click a link or receive a download from a suspect email, you must report it as an incident following the steps above. Successful phishing attacks are security incidents and should be reported immediately. Phishing emails that are automatically routed to your spam folder do not need to be reported. Your vigilance also helps CivicActions IT to prepare against similar phishing attacks that might be sent to other team members.
+
+## Reporting other incidents
+To report a security incident, follow *all* of the steps below:
+
+1. Send an email to [support@civicactions.com](mailto:support@civicactions.com) within 1 hour of identifying an incident. Please include *Security Incident* in the subject line, along with a brief description of the incident (Ex. security token committed to Gitlab repo). Don't worry if you don't have all of the details gathered when you email CivicAction's Incident Response (IR) team. The critical piece here is notification within one hour.
```

Comment:
It's basically the #it_is team (Owen, David, Josh, Karen, Marc and myself). In the LINCS incident response ticket, I include a link to the CivicActions/LINCS IR team (https://docs.google.com/a/civicactions.net/spreadsheets/d/1fhjOq6KpURCfplu6vbIAV1HbQRO-7jUqX_Uj8gE1vC0/edit?usp=sharing). I'd like to set this up in a sheet that is CivicActions only with each person tagged with the project teams they are specifically attached to, so that the project team contact lists can pull in the CivicActions team members from that central list. There's probably a way to make this happen...